classes_assignment.js

```diff
@@ -0,0 +1,61 @@
+// class Human {
+//   constructor(name, age){
+//     this.name = name;
+//     this.age = age;
+//   }
+// ageOneYear() {
+//   return this.age++
+// }
+//
+// eating(str) {
+//   return "mmm, mmm, mmm, I\'m love\'n it"
+// }
```

Comment:
For your eating function:
- No argument is needed
- We want use console.log on line 11 (instead of return) so that when the function is called upon, it will log the line instead of returning it